simapp/app.go

```diff
@@ -302,6 +302,7 @@ func NewSimApp(
 	)
 
 	app.CircuitKeeper = circuitkeeper.NewKeeper(keys[circuittypes.StoreKey], authtypes.NewModuleAddress(govtypes.ModuleName).String(), app.AccountKeeper.AddressCodec())
+	app.BaseApp.SetCircuitBreaker(&app.CircuitKeeper)
```

Comment:
Can you add circuit's module.go `ProvideModule` function, and add `runtime.BaseAppOption` as an outputs?

```go
	baseappOpt := func(app *baseapp.BaseApp) {
		app.SetCircuitBreaker(&k)
	}
```

Then this will be done automatically for the user of app config.

Reply:
done thanks